app/controllers/api_controller/arbitration_profiles.rb

```diff
@@ -0,0 +1,44 @@
+class ApiController
+  module ArbitrationProfiles
+    def create_resource_arbitration_profiles(_type, _id, data)
+      validate_profile_data(data)
+      attributes = data.dup
```

Comment:
Just wondering if this can now be DRY'd up? Something like a `build_arbitration_attributes` method or similar?